### PR TITLE
Serenity updates - September - Rules UI completed

### PR DESCRIPTION
**Pull request by morganen**

### CODE REVIEW

**Review by NolwennP — COMMENTED** [review]
Maybe to celebrate the completion of this screen, put a screenshot of the interface. What do you think @LaureBro?

File: content/updates/2020-09/1-rules-UI-complete.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+pim_announcement_audience:
+- EE
+---
+
+# The rules builder interface 100% complete!
```
Comment (NolwennP) [inline content/updates/2020-09/1-rules-UI-complete.md:6]:
```suggestion
# The rules engine UI 100% complete!
```

File: content/updates/2020-09/1-rules-UI-complete.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+pim_announcement_audience:
+- EE
+---
+
+# The rules engine UI 100% complete!
```
Comment (NolwennP) [inline content/updates/2020-09/1-rules-UI-complete.md:6]:
```suggestion
# The Rules Builder 100% complete!
```
```suggestion
# The rules engine UI 100% complete!
```

File: content/updates/2020-09/1-rules-UI-complete.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+---
+pim_announcement_audience:
+- EE
+---
+
+# The Rules Builder 100% complete!
```
Comment (LaureBro) [inline content/updates/2020-09/1-rules-UI-complete.md:6]:
```suggestion
# The rules engine user interface 100% complete!
```

File: content/updates/2020-09/1-rules-UI-complete.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+::: meta-data type="New" features="Rules engine, Productivity" available="late September" link-to-doc="../articles/manage-your-rules.html#the-rule-builder-tab"
+:::
+
+You can now **entirely** create your rules from our PIM interface. Indeed, the rules builder has been updated with all the actions and conditions available to build a rule.
```
Comment (LaureBro) [inline content/updates/2020-09/1-rules-UI-complete.md:10]:
```suggestion
The Rules Builder is now compatible with all the rules engine's conditions and actions, so you can now **entirely** create your rules from your PIM user interface. 
```

File: content/updates/2020-09/1-rules-UI-complete.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+It means that you don't need to use your old YAML files anymore. You can very easily build and maintain all your rules directly from within this powerful interface.
```
Comment (LaureBro) [inline content/updates/2020-09/1-rules-UI-complete.md:12]:
```suggestion
If you were not comfortable with YML files, the rules builder screen will allow you to easily create and manage your rules directly from your PIM! 
```

File: content/updates/2020-09/1-rules-UI-complete.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+- EE
+---
+
+# The rules engine user interface 100% complete!
+::: meta-data type="New" features="Rules engine, Productivity" available="late September" link-to-doc="../articles/manage-your-rules.html#the-rule-builder-tab"
+:::
+
+The Rules Builder is now compatible with all the rules engine's conditions and actions, so you can now **entirely** create your rules from your PIM user interface. 
+
+If you were not comfortable with YML files, the rules builder screen will allow you to easily create and manage your rules directly from your PIM!
```
Comment (NolwennP) [inline content/updates/2020-09/1-rules-UI-complete.md:12]:
```suggestion
If you were not comfortable with YAML files, the rules builder screen will allow you to easily create and manage your rules directly from your PIM! 
```

**Review by LaureBro — APPROVED** [review]

**Pull request merged**